streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -790,7 +790,7 @@ public KafkaStreams(final Topology topology,
     public KafkaStreams(final Topology topology,
                         final Properties props,
                         final Time time) {
-        this(topology, new StreamsConfig(props), time);
+        this(topology, new InternalStreamsConfig(props), null, time);
```

Comment:
Some side cleanup on constructor methods -- no public change -- just to make the flow simpler